moto/apigateway/responses.py

```diff
@@ -328,31 +335,38 @@ def integrations(self, request, full_url, headers):
         method_type = url_path_parts[6]
 
         try:
+            integration_response = {}
+
             if self.method == "GET":
                 integration_response = self.backend.get_integration(
                     function_id, resource_id, method_type
                 )
             elif self.method == "PUT":
                 integration_type = self._get_param("type")
                 uri = self._get_param("uri")
-                integration_http_method = self._get_param("httpMethod")
-                creds = self._get_param("credentials")
+                credentials = self._get_param("credentials")
                 request_templates = self._get_param("requestTemplates")
+                method = self.backend.get_method(function_id, resource_id, method_type)
```

Comment:
If I'm reading this correctly, 'method' doesn't do anything - is this just to verify that the method exists? If so, please add a comment